src/main/java/io/openliberty/tools/common/plugins/util/DevUtil.java

```diff
@@ -1256,7 +1259,17 @@ private String readStdOut(Process p) throws IOException, InterruptedException {
     private String getContainerCommand() {
         StringBuilder command = new StringBuilder("docker run --rm");
         if (!skipDefaultPorts) {
-            command.append(" -p 9080:9080 -p 9443:9443");
+            int httpPortToUse, httpsPortToUse;
+            try {
+                httpPortToUse = findAvailablePort(LIBERTY_DEFAULT_HTTP_PORT, false);
+                httpsPortToUse = findAvailablePort(LIBERTY_DEFAULT_HTTPS_PORT, false);
+            } catch (IOException x) {
+                error("An error occurred while trying to find a free network port, using default port numbers.", x);
```

Comment:
`error("An error occurred while trying to find an available network port. Using default port numbers.", x);`

Reply:
😎👍